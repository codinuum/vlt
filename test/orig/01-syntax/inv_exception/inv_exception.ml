let () = [%LOG [%DEBUG] "msg" [%EXCEPTION 10]]
