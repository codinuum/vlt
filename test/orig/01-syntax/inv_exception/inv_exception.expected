File "inv_exception.ml", line 1, characters 42-44:
1 | let () = [%LOG [%DEBUG] "msg" [%EXCEPTION 10]]
                                              ^^
Error: This expression has type int but an expression was expected of type
         exn
